react/features/prejoin/middleware.js

```diff
@@ -29,6 +31,15 @@ declare var APP: Object;
  */
 MiddlewareRegistry.register(store => next => async action => {
     switch (action.type) {
+    case CONFERENCE_JOINED: {
+        const { getState, dispatch } = store;
+        const { enableForcedReload } = getState()['features/base/config'];
+
+        dispatch(setPrejoinPageVisibility(false));
```

Comment:
Nope, this is needed because prejoin page visibility is set to false only when conference join happened from pre-join page. After the forced reload, if the prejoin page visibility is not toggled, the UI shows the pre-join page even the user has joined the conference automatically.
The prejoin page is not disabled, its only skipped for this particular reload.